custom/icds_reports/utils/aggregation_helpers/distributed/thr_form_v2.py

```diff
@@ -36,9 +36,8 @@ def aggregation_query(self):
         }
 
         return """
-        INSERT INTO "{tablename}" (
-        state_id, supervisor_id, awc_id, thr_distribution_image_count, month
-        ) (
+        DROP TABLE IF EXISTS "temp_thr";
```

Comment:
Hey, should not this use DROP TABLE instead of delete [here](https://github.com/dimagi/commcare-hq/pull/24829/files#diff-018341eed6714edf79341ff3e1cc2ceeR22) if we are changing to this(Refering to [this](https://github.com/dimagi/commcare-hq/blob/master/custom/icds_reports/utils/aggregation_helpers/distributed/ls_vhnd_form.py#L20))?

Reply:
it is a temporary table so it gets recreated each time, since it only lasts for the duration of the query

Reply:
No, i was talking about the main table not the temp table. 
Is it intentional that we are not using child table thing here with "DROP table query" for main table instead of just using the main table to insert into it with "DELETE query"?
Wont this delete query in `drop_table_query` would start the autovaccum problem which happend in a couple of weeks back? @calellowitz

Reply:
it should not matter. this table is significantly smaller and we are deleting and replacing a much smaller percentage of the overall table (as opposed to the thing we saw before where the whole table was replaced). And to that point we can't drop it because we are not replacing the whole table, we need to preserve some data.